Automattic/woocommerce-payments
# Fix date type in create payment intent response schema

Fixes 286-gh-Automattic/woocommerce-payments-meta 

#### Changes proposed in this Pull Request

Display formatted created date for payment intent endpoint.









#### Testing instructions





* set the capture later option for the store
* create an order
* using get order API, get the order details ( GET  <URL>/wp-json/wc/v3/orders/<order_id>)
* using the create payment intent API, create a new payment intent for the order 
* URL: POST <URL>/wp-json/wc/v3/payments/payment_intents
* Body: 
 ```
{
	"payment_method":"pm_....",
	"customer":"cus_....",
	"order_id":"xxxxx"
}
 ```
* check the response. `created` should be formatted instead of timestamp
-------------------


**Post merge**
